ddtrace/internal/tracemethods.py

```diff
@@ -1,11 +1,17 @@
-import typing  # noqa:F401
+from typing import List
 
 import wrapt
 
+from ddtrace.internal.logger import get_logger
 
-def _parse_trace_methods(raw_dd_trace_methods):
-    # type: (str) -> typing.List[str]
-    """Return the methods to trace based on the specification of DD_TRACE_METHODS.
+
+log = get_logger(__name__)
+
+
+def _parse_trace_methods(raw_dd_trace_methods: str) -> List[str]:
```

Comment:
Any chance the syntax can be changed to `module.path:func_qualname`? This way we would know exactly on which module to put the hook, and what to look for within that module when it is loaded. We could then use the `inspect` module to determine if the function is a coro/async/generator etc...

Reply:
So in the case we have 2 functions from the same module, would `raw_dd_trace_methods` be formatted as `"mymodule.mysubmodule:myclass.myfunc,mymodule.mysubmodule:myclass.otherfunc"`? The team has also proposed something like: `"mymodule.mysubmodule[myclass.myfunc, myclass.otherfunc]"`